src/base/views/mouse-tool.ts

```diff
@@ -190,6 +238,10 @@ export class MouseListener {
         return [];
     }
 
+    wheels(target: SModelElement[], events: WheelEvent[]): (Action | Promise<Action>)[] {
```

Comment:
To avoid breaking the code of users not having implemented the wheels function yet, this should have a default implementation repeatedly calling the wheel function, collecting and returning the actions.

Reply:
I made the change for now but this is not as easy as it seems...
The wheel function might use the ```viewport = findParentByFeature(targets[i], isViewport);``` to access the current viewport but since the viewport wont change between the executed wheel actions (when exec from wheels) it will never change and might therefor break users code nonetheless